src/Reflector.php

```diff
@@ -14,10 +14,10 @@
 
 namespace Humbug\PhpScoper;
 
+use JetBrains\PHPStormStub\PhpStormStubsMap;
```

Comment:
I think StyleCI is wrong

Reply:
It applies the `ordered_use` fixer: https://docs.styleci.io/fixers#alpha_ordered_imports